src/main/java/com/quizte/mapper/UserMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.quizte.mapper;
+
+import com.quizte.dto.RegistrationUserDto;
+import com.quizte.entity.User;
+import org.modelmapper.ModelMapper;
+
+public class UserMapper {
+    private static final ModelMapper mapper = new ModelMapper();
+
+    private UserMapper() {
+    }
+
+    public static User toUser(RegistrationUserDto registrationUserDto) {
```

Comment:
Don't use static methods apart from general utility methods, such as date formatting, string processing, etc. for anything else - use spring components